<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PineSaveGameEditorClass</class>
 <widget class="QMainWindow" name="PineSaveGameEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PineSaveGameEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>0</y>
      <width>475</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No directory chosen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Game Location:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="keyEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>200</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSelect_Save_Directory"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSelect_Save_Directory">
   <property name="text">
    <string>Select Save Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PineSaveGameEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSelect_Save_Directory</sender>
   <signal>triggered()</signal>
   <receiver>PineSaveGameEditorClass</receiver>
   <slot>SaveGameLocationChooser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>PineSaveGameEditorClass</receiver>
   <slot>OpenSaveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SaveGameLocationChooser()</slot>
  <slot>OpenSaveFile()</slot>
 </slots>
</ui>
